hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -387,17 +387,54 @@
     },
 
     { name: "SIDELOAD_CLEAR",
-      desc: "Clear all sideload keys",
+      desc: "sideload key slots clear"
       swaccess: "rw",
       hwaccess: "hro",
       fields: [
-        { bits: "0",
+        { bits: "2:0",
           name: "VAL",
           resval: "0"
           desc: '''
-            When 1, the sideload keys are continuously cleared with entropy.
-            When 0, the sideload keys are left as is.
-          '''
+            Depending on the value programmed, a different side load key slot is cleared.
+            If the value programmed is not one of the enumerated values below, ALL side load key slots are continuously cleard.
+          ''',
+          enum: [
+            { value: "0",
+              name: "None",
+              desc: '''
+                No sideload keys cleared.
+                '''
+            },
+            { value: "1",
+              name: "AES",
+              desc: '''
+                When 1, the AES sideload key is continuously cleared with entropy.
+                When 0, the AES sideload key is left as is.
```

Comment:
Just nitpicks. `When 1` is not appropriate other than this field. HMAC/ KMAC/ OTBN case, SW should write the value `2/ 3/ 4` right?

Reply:
o shoot good catch.